packages/create-platformatic-db/help/help.txt

```diff
@@ -0,0 +1,10 @@
+Welcome to Platformatic DB Creator. 
```

Comment:
Since we have a prompt, CLI commands need only to create an app automatically without any questions. So IMO, it should be possible to create an app without any questions.

Reply:
Not sure what you mean here, can you please elaborate? (maybe with an example)?

Reply:
I would like to be able to create an application using only CLI keys. Without any question. Otherwise, I don't see the case when CLI keys can be useful.

Reply:
OK, but you can always uses `db init`. Also, I am going to change this to create also `service` (and in the future the other stuff). So the set of options (and questions) will change accordingly with what we are going to create. 
IMO we should have: 
- the `cli` commands (now in `db` and `service` [WIP]) to create without interactions
- this (renamed `create-platformatic`) which is interactive, using the code of `db` and `service` (and the future components) as wizard to create everything.  
What do you think?

Reply:
Hmm, do we still want to have the init commands after this PR?

Reply:
I don't think it will, if one (this) uses the other (db/init).